libretro/libretro-mapper.c

```diff
@@ -1170,14 +1170,16 @@ void update_input(int disable_physical_cursor_keys)
 
 void retro_poll_event()
 {
-   /* If RetroPad is controlled with keyboard keys, then prevent up/down/left/right/fire/fire2 from generating keyboard key presses */
+   /* If RetroPad is controlled with keyboard keys, then prevent up/down/left/right/b/a/x/y from generating keyboard key presses */
    if (uae_devices[0] == RETRO_DEVICE_JOYPAD && ALTON==-1 && 
       (input_state_cb(0, RETRO_DEVICE_JOYPAD, 0, RETRO_DEVICE_ID_JOYPAD_B) ||
-       input_state_cb(0, RETRO_DEVICE_JOYPAD, 0, RETRO_DEVICE_ID_JOYPAD_A)
+       input_state_cb(0, RETRO_DEVICE_JOYPAD, 0, RETRO_DEVICE_ID_JOYPAD_A) ||
```

Comment:
Doesn't this prevent e.g. mapping a keyboard key to button X?

Reply:
What I mean is: "Doesn't this prevent me from using mapper to map button X to act as a keyboard key"?

I am probably wrong.

Reply:
I am wrong.

Reply:
Aye, f.ex if RetroPad X is pressing Space via mapper and Space is RetroPad X, with pass-through enabled Space will be pressed twice, and now only once with pass-through disabled.